server/app.js

```diff
@@ -6,7 +6,7 @@ const mongoose = require('mongoose')
 
 const app = express()
 
-const userRoutes = require('./api/routes/user')
+const indexRoutes = require('./api/routes/')
```

Comment:
rename to routes?

Reply:
we dont need to add file name if it's index .It's the same as - 
...=require('./api/routes/index')

Reply:
I meant can we rename `indexRoutes` to just `routes`?